cli/azd/extensions/microsoft.ai.agents/internal/cmd/context.go

```diff
@@ -0,0 +1,155 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/arm"
+	"github.com/azure/azure-dev/cli/azd/pkg/azdext"
+	"github.com/fatih/color"
+	"github.com/spf13/cobra"
+)
+
+func newContextCommand() *cobra.Command {
+	return &cobra.Command{
+		Use:   "context",
+		Short: "Get the context of the AZD project & environment.",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			// Create a new context that includes the AZD access token
+			ctx := azdext.WithAccessToken(cmd.Context())
+
+			// Create a new AZD client
+			azdClient, err := azdext.NewAzdClient()
+			if err != nil {
+				return fmt.Errorf("failed to create azd client: %w", err)
+			}
+
+			defer azdClient.Close()
+
+			hasEnv := false
+
+			getConfigResponse, err := azdClient.UserConfig().Get(ctx, &azdext.GetUserConfigRequest{
+				Path: "",
+			})
+			if err == nil {
```

Comment:
Is it okay to fall through if any of the statements in the 'if' body fail? If so, a comment would be good, just to state it's intentional and that it doesn't matter if we actually get a valid JSON user config.

Reply:
@wbreza is this supposed to print the entire `~/.azd/config.json` as JSON? If so it may not be working correctly when you pass `Path: ""` 

```go
if err == nil {
	if getConfigResponse.Found {
		color.HiWhite("User Config")
		var userConfig map[string]string
		err := json.Unmarshal(getConfigResponse.Value, &userConfig)
		if err == nil {
			jsonBytes, err := json.MarshalIndent(userConfig, "", "  ")
			if err == nil {
				fmt.Println(string(jsonBytes))
			}
		}
	}
}
```